.github/workflows/ruby_on_rails.yml

```
@@ -1,9 +1,24 @@
-name: Ruby on Rails
+name: CI
 
 on: [push, pull_request]
 
 jobs:
-  build:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.7.x
+    - name: Build and test with Rake
```

Should we rename this?
```suggestion
    - name: Lint with Rubocop
```